lambdas/search/index.py

```diff
@@ -37,11 +71,6 @@ def lambda_handler(request):
         connection_class=RequestsHttpConnection
     )
 
-    index = request.pathParameters['proxy']
-    body = request.args.get('source')
-    _source = request.args.get('_source')
-    size = request.args.get('size', '1000')
-
     result = es_client.search(index, body, _source=_source, size=size, timeout=MAX_QUERY_DURATION)
```

Comment:
Oh, for this we should probably have a global override-able by the environment so that we can skip indexes that are under load or incomplete. So:
```
INDEX_OVERRIDES = os.environ.get("INDEX_OVERRIDES") or "-elevation-tiles-prod*"
...
indexes = index + "," + INDEX_OVERRIDES
```
Keep in mind that "index" can be comma-separated list; ditto for INDEX_OVERRIDES.

Reply:
Note the `-` in front of `the index name.

Reply:
Ok; I'll do that in a separate PR